### PR TITLE
[chore] Move all instance (model) functions to services (part 3)

### PR DESCRIPTION
**Pull request by nwithan8**
# Description

Subpart of #441 , see for more details

# Testing

- Unit tests moved, re-recorded as needed

# Pull Request Type

Please select the option(s) that are relevant to this PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement (fixing a typo, updating readme, renaming a variable name, etc)

### CODE REVIEW

File: EasyPost.Tests/BetaFeaturesTests/ServicesTests/WebhookServiceTest.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public async Task TestCreate()
         {
             UseVCR("create");
 
-            const string url = "https://example.com/create";
+            string url = $"https://example.com/beta/create/{TestUtils.NetVersion}";
```
Comment (Justintime50) [inline EasyPost.Tests/BetaFeaturesTests/ServicesTests/WebhookServiceTest.cs:43]: question: why is `beta` in here?

File: EasyPost.Tests/BetaFeaturesTests/ServicesTests/WebhookServiceTest.cs [hunk 2]
```diff
@@ -73,6 +73,32 @@ public async Task TestAll()
...
         }
 
+        [Fact]
+        [CrudOperations.Update]
+        [Testing.Function]
+        public async Task TestUpdate()
+        {
+            UseVCR("update");
+
+            string url = $"https://example.com/beta/update/{TestUtils.NetVersion}";
```
Comment (Justintime50) [inline EasyPost.Tests/BetaFeaturesTests/ServicesTests/WebhookServiceTest.cs:83]: question: why is `beta` in here?

File: EasyPost.Tests/EasyPost.Tests.csproj [hunk 3]
```diff
@@ -65,9 +65,77 @@
     <None Remove="RestSharp" />
     <None Remove="MSTest.TestFramework" />
     <None Remove="MSTest.TestAdapter" />
+    <None Remove="cassettes\net\beta_rate_service\get_lowest_rate_instance_function.json" />
```
Comment (Justintime50) [inline EasyPost.Tests/EasyPost.Tests.csproj:68]: question: why is this here? Would prefer we don't need to maintain this brittle list which is likely to change.

**Review by Justintime50 — CHANGES_REQUESTED** [review]

File: EasyPost.Tests/BetaFeaturesTests/ServicesTests/WebhookServiceTest.cs (hunk 2 above)
Comment (nwithan8) [reply EasyPost.Tests/BetaFeaturesTests/ServicesTests/WebhookServiceTest.cs:83]: Since this is testing in the beta features for the parameters set. This is to avoid a server-side conflict re: duplicate webhooks when running the test suite.

File: EasyPost.Tests/EasyPost.Tests.csproj (hunk 3 above)
Comment (nwithan8) [reply EasyPost.Tests/EasyPost.Tests.csproj:68]: Didn't mean to have this included, rolling back.

**Review by Justintime50 — APPROVED** [review]

**Review by jchen293 — APPROVED** [review]
